api_documentation_v1.yaml

```diff
@@ -0,0 +1,406 @@
+          schema:
+            type: integer
+            format: int64
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/RecipeDailyReportExternalDTO'
+        '401':
+          description: Unauthorized
+        '403':
+          description: Forbidden
+        '404':
+          description: Not Found
+      security:
+        - Static token:
+            - global
+  /api/v1/external/ration:
+    post:
+      tags:
+        - report-external-applications-controller
+      summary: importExternalRation
+      operationId: importExternalRationUsingPOST
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: array
+              items:
+                $ref: '#/components/schemas/ExternalRationRequestDTO'
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+        '201':
+          description: Created
+        '401':
+          description: Unauthorized
+        '403':
+          description: Forbidden
+        '404':
+          description: Not Found
+      security:
+        - Static token:
+            - global
+components:
+  schemas:
+    ExternalRationIngredientRequestDTO:
+      title: ExternalRationIngredientRequestDTO
+      type: object
+      properties:
+        amountAsFed:
+          type: number
+          description: Amount as fed of ingredient in ration
+          format: double
+        amountDm:
+          type: number
+          description: Amount DM of ingredient in ration
+          format: double
+        costAsFed:
+          type: number
+          description: Cost of amount as fed of ingredient in ration
+          format: double
+        costDm:
+          type: number
+          description: Cost DM of ration ingredient
+          format: double
+          example: 41.25
+        dm:
+          type: number
+          description: DM percentage of ration ingredient
+          format: double
+        externalId:
+          type: integer
+          description: >-
+            ExternalId represent identification number of ration ingredient
+            between systems
+          format: int64
+          example: 965
+        feedName:
+          type: string
+          description: Name of ration ingredient
+          example: Sojabohnen (gesch., Bioprofin)
+        feedType:
+          type: string
+          description: Feed type of ingredient in ration
+          example: >-
+            Dry forage,Corn silage,Alfalfa silage, Grass silage, Other wet
+            forage, By-productGrain ,Fatty acid source ,Triglycerides, Cp
+            source, Rup source, Minerals, Miscellaneous, Vitamin, Premix,
+            Forage, Concentrate, Commercial
+        orderNumber:
```

Comment:
We have this information only in the reports, representing the real feeding times. The users can also define planned feeding times on a pen level (when they create a pen), but from our knowledge, they usually don't follow those times.

Reply:
ok, I don't mean feeding-times. I mean the time the farmer has to wait between loading different ingredients into the wagon because the stirring screw needs some seconds/minutes until everything is mixed together - there is also an upper limit that the screw shouldn't mix too long because than the ration degrades. I know at least two vendors which have that as property of a loading-ingredient but I haven't found such settings in the mmmooogle interface. do you have a default here?